File: Assignment no. 59/Q1.java
import java.util.Scanner;

class Q1
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.println("Enter the number of elements:");
		int Arr[]=new int[sobj.nextInt()];
		System.out.println("Enter the elements:");
		for(int i=0;i<Arr.length;i++)
		{
			Arr[i]=sobj.nextInt();
		}
		System.out.println("Enter the number");
		int no=sobj.nextInt();
		Num obj=new Num();
		if(obj.check(Arr,no))
		{
			System.out.println("Number is present in array");
		}
		else
		{
			System.out.println("Number is not present in array");
		}
	}
}

File: Assignment no. 23/Q1.c
#include<stdio.h>
#include<stdlib.h>
typedef int BOOL;
#define TRUE 1
#define FALSE 0
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//  Name        :   Check
//  Input       :   integer address,integer,integer
//  Output      :   boolean
//  Description :   Check weather number is present in given array
//  Author      :   Nikhil
//  Date        :   03/10/2019
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

BOOL Check(int Arr[],int iLength,int iNo)
{
    int iCnt=0;
    if((NULL==Arr) || (iLength<=0))
    {
        return FALSE;
    }
    for(iCnt=0;iCnt<iLength;iCnt++)
    {
        if(Arr[iCnt]==iNo)
        {
            break;
        }
    }
    if(iCnt==iLength)
    {
        return FALSE;
    }
    else
    {
        return TRUE;
    }
}

//Entry point function

void main()
{
    int iSize=0,iCnt=0,iValue=0;
    BOOL iRet=FALSE;
    int *p=NULL;
    printf("Enter number of elements:\n");
    scanf("%d",&iSize);
    p=(int *)malloc(iSize*sizeof(int));
    if(NULL==p)
    {
        printf("Unable to allocate memory");
        return -1;
    }
    for(iCnt=0;iCnt<iSize;iCnt++)
    {
        printf("Enter element : %d:\n",iCnt+1);
        scanf("%d",&p[iCnt]);
    }
    printf("Enter number:\n");
    scanf("%d",&iValue);
    iRet=Check(p,iSize,iValue);
    if(iRet==TRUE)
    {
        printf("TRUE");
    }
    else
    {
        printf("FALSE");
    }
    free(p);
    return 0;
}


